Nasal/electrical.nas

```diff
@@ -495,18 +495,26 @@ var avionics_bus_1 = func() {
     return load;
 }
 
+############################ Utility function
+
+var flapsDown = controls.flapsDown;
```

Comment:
Where does this code come from?

Reply:
It seems we used to have an override, but removed it in #1236

Reply:
(@wlbragg so this change should be removed otherwise we undo the fix of #1236)

Reply:
@Zaretto indicated this was the proper way to do an overload of a base function in controls, see: https://github.com/c172p-team/c172p/issues/1265#issuecomment-484159150 Although where is the "flapsUp". @gilbertohasnofb  did that commit so I need to let him sort if out if possible.

Reply:
> @Zaretto indicated this was the proper way to do an overload of a base function in controls, 

That's correct; however it's not necessarily the best way to do what that code is doing and a listener on `controls/flight/flaps` is probably a better way to do it and will handle up and down.

Overriding the methods in $FGData/Nasal/controls.nas is really more suited to prevent a control from being moved due to aircraft specific interlocks. For example on the B737 you can't physically move the gear lever when on the ground (unless you pull a little lever underneath the handle).

Reply:
@wlbragg But @Zaretto used fgdata. If you check [`master`](https://github.com/c172p-team/c172p/blob/master/Nasal/electrical.nas) you see there's no longer any override, so why introduce it again?